### PR TITLE
[WIP] Stop skipping tests

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/TelephoneItem.php [hunk 1]
```diff
@@ -17,7 +19,9 @@ public static function generateSampleValue(FieldDefinitionInterface $field_defin
     $values = parent::generateSampleValue($field_definition);
     switch ($field_definition->getName()) {
       case 'field_block_phone_number':
-        $values['value'] = '+' . $values['value'];
+        $phoneNumberUtil = PhoneNumberUtil::getInstance();
```
Comment (thewilkybarkid) [inline src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/TelephoneItem.php]: Should use DI, but will need to work out how to do that.

File: src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/TelephoneItem.php [hunk 2]
```diff
@@ -17,7 +19,9 @@ public static function generateSampleValue(FieldDefinitionInterface $field_defin
...
+        $number = $phoneNumberUtil->getExampleNumber(1);
```
Comment (thewilkybarkid) [inline src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/TelephoneItem.php]: Not random.

**Comment by thewilkybarkid** [comment]
This seems like a rabbit hole.

**Comment by giorgiosironi** [comment]
This is best-effort, was more meant to enable the tests that now pass (if any) or can be made to pass easily given the data generators that are now in use.

**Comment by giorgiosironi** [comment]
From in-person conversation: this is likely to get outdated due to ckeditor evolution.

**Comment by nlisgo** [comment]
Have addressed the skipped tests in the following PR: https://github.com/elifesciences/journal-cms/pull/499

Of course, it's not ideal that the data is random.

**Pull request closed without merging**